BrainCloudClient/tests/TestLobby.cs

```diff
@@ -143,18 +143,65 @@ public void TestUpdateSettings()
 
             tr.RunExpectFail(StatusCodes.BAD_REQUEST, ReasonCodes.LOBBY_NOT_FOUND);
         }
+        
+        [Test]
+        public void TestGetLobbyInstance()
+        {
+            TestResult tr = new TestResult(_bc);
+
+            Dictionary<string, int> ratings = new Dictionary<string, int>();
+            ratings["min"] = 50;
+            ratings["max"] = 70;
+            
+            Dictionary<string, int> ping = new Dictionary<string, int>();
```

Comment:
We don't need ping for that test. It will be ignored. You can remove it.

Reply:
The criteriaJson is looking for the ping within the message or else it will return with:
`Processing exception (message): Required criteriaJson parameter 'ping' is missing."`